### ISSUE TITLE
Add YARA command-line options to Settings

### ISSUE THREAD

**Issue by infosec-intern** [infosec-intern/vscode-yara#10]
YARA Help Menu
```
YARA 3.5.0, the pattern matching swiss army knife.
Usage: yara [OPTION]... RULES_FILE FILE | DIR | PID

Mandatory arguments to long options are mandatory for short options too.

  -t,  --tag=TAG                   print only rules tagged as TAG
  -i,  --identifier=IDENTIFIER     print only rules named IDENTIFIER
  -n,  --negate                    print only not satisfied rules (negate)
  -D,  --print-module-data         print module data
  -g,  --print-tags                print tags
  -m,  --print-meta                print metadata
  -s,  --print-strings             print matching strings
  -e,  --print-namespace           print rules' namespace
  -p,  --threads=NUMBER            use the specified NUMBER of threads to scan a directory
  -l,  --max-rules=NUMBER          abort scanning after matching a NUMBER of rules
  -d VAR=VALUE                     define external variable
  -x MODULE=FILE                   pass FILE's content as extra data to MODULE
  -a,  --timeout=SECONDS           abort scanning after the given number of SECONDS
  -k,  --stack-size=SLOTS          set maximum stack size (default=16384)
  -r,  --recursive                 recursively search directories
  -f,  --fast-scan                 fast matching mode
  -w,  --no-warnings               disable warnings
  -v,  --version                   show version information
  -h,  --help                      show this help and exit

Send bug reports and suggestions to: vmalvarez@virustotal.com.
```

YARAC Help Menu
```
Usage: yarac [OPTION]... [NAMESPACE:]SOURCE_FILE... OUTPUT_FILE

  -d VAR=VALUE           define external variable
  -w,  --no-warnings     disable warnings
  -v,  --version         show version information
  -h,  --help            show this help and exit

Send bug reports and suggestions to: vmalvarez@virustotal.com.
```

**Comment by infosec-intern** [comment]
Looks like the Yara object won't reread settings once its been loaded. Looking for some kind of OnDidSettingsChange function to register a callback

**Comment by infosec-intern** [comment]
Found OnDidChangeConfiguration function, but i have not added support for improper flags, so YARA rules will appear to compile successfully when the process just errored out early

**Comment by infosec-intern** [comment]
Fixed in 6b675ca7b2197c7b9161b5bd7e74078aa081ce0e
